src/components/common/header.js

```diff
@@ -12,6 +12,15 @@ export const Header = ({ setShowAdmin }) => {
   const [isSignedIn, setIsSignedIn] = React.useState(false);
   const [isAdmin] = useAdmin({ address: wallet.accountId });
 
+  //When user login with default wallet id it will contain 64 characters, so this function will show first 6 and last 4 chars
+  const formatAccountId = (accountId) =>
+    accountId.includes('.')
+      ? accountId
+      : `${accountId.substring(0, 6)}...${accountId.substring(
+          accountId.length - 4,
+          accountId.length
+        )}`;
+
```

Comment:
I think instead of just showing the first few and last few characters only, we can have it dynamic depending upon the screen width, since most of the times we will have the user having their name as the account ID and it makes sense to show them the full address

Reply:
Here I am checking if user login with name it will contain dot ( like xxx.near) so I will not format it ( line 17 ), in all other case it will have 64 characters so only in that case I will format it. 
But if you want I can use break-word and split 64 chars in 2 lines on mobile and user can see whole address. 
Feel free to tell me which one is better for this use case and I will refactor it.

Reply:
@Megha-Dev-19 Please reply we need to close this